backend/src/main/java/se/chalmers/dat265/group1/storage/PostgresSQLConnector.java

```diff
@@ -0,0 +1,56 @@
+package se.chalmers.dat265.group1.storage;
+
+import java.sql.*;
+import java.util.Properties;
+import org.apache.commons.logging.*;
+
+public class PostgresSQLConnector implements DBInterface {
+    private Log log = LogFactory.getLog(PostgresSQLConnector.class);
+    private static final String CONNECTION_URL_DOCKER = "jdbc:postgresql://db:5432/swereadb";
+    private static final String CONNECTION_URL_DEBUG = "jdbc:postgresql://localhost:5433/swereadb";
+    private static final String USERNAME = "admin";
+    private static final String PWD = "1234";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'PWD' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)